docs/sphinx/data_center_install.rst

```diff
@@ -0,0 +1,371 @@
+  `OPENSSL_INSTALL_PREFIX` variable to, you can omit setting the FC 
+  environment variable.
+- To use GPU-acceleration in CUDA Quantum, make sure to set CUDACXX to 
+  your CUDA compiler. If the CUDA compiler is not found when building
+  CUDA Quantum, some components and backends will be omitted automatically 
+  during the build.
+
+Building CUDA Quantum
+------------------------------------
+
+This installation guide has been written for a specific version/commit of CUDA Quantum. 
+Make sure to obtain the source code for that version. 
+Clone the CUDA Quantum `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`__ and
+checkout the appropriate branch, tag, or commit. 
+Note that the build scripts assume that they are run from within a git repository, 
+and merely downloading the source code as ZIP archive hence will not work.
+
+From within the folder where you cloned the CUDA Quantum repository, run the following
+command to build CUDA Quantum:
+
+.. code-block:: bash
+
+    .. literalinclude:: docker/build/assets.Dockerfile
+      :start-after: [>CUDAQuantumBuild]
+      :end-before: [<CUDAQuantumBuild]
+
+The CUDA Quantum build will compile or omit optional components automatically depending
+on whether the necessary pre-requisites are found in the build environment.
+Please check the build log to confirm that all desired components have been built. If
+you see a message that a component has been skipped, make sure you followed the 
+instructions for installing the necessary prerequisites and build dependencies, 
+and have set the necessary environment variables as described in this document.
+
+Preparing the Installation
+------------------------------------
+
+To easily migrate the built binaries to the host system, we recommend creating a
+`self-extracting archive <https://makeself.io/>`__. To do so, download the 
+`makeself script(s) <https://github.com/megastep/makeself>`__ and move the necessary 
+files to install into a separate folder using the command
+
+.. code-block:: bash
+
+    .. literalinclude:: docker/release/installer.Dockerfile
+      :start-after: [>CUDAQuantumAssets]
+      :end-before: [<CUDAQuantumAssets]
+
+You can then create a self-extracting archive with the command
+
+.. code-block:: bash
+
+    ./makeself.sh --gzip --license cuda_quantum_assets/cudaq/LICENSE \
+        cuda_quantum_assets install_cuda_quantum.$(uname -m) \
+        "CUDA Quantum toolkit for heterogeneous quantum-classical workflows" \
+        bash cudaq/migrate_assets.sh -t /opt/nvidia/cudaq
+
+Installation on the Host
+------------------------------------
+
+Make sure your host system satisfies the `Prerequisites`_ listed above,
+and that the same C++ standard library that was used during the
+CUDA Quantum build is present and discoverable on the host system. 
+While not strictly necessary, we recommend that you install the toolchain 
+that was used for the CUDA Quantum build on the host.
```

Comment:
Does the self-installer grab the LLVM installation and deploy it with the CUDA Quantum installation?

Reply:
Yes, it does, but the clang compiler that will be installed along with nvq++ is not added to the PATH and hence usable only if you know where to look (one could look at the build configuration in the CUDA_QUANTUM_PATH folder to learn where LLVM was installed, and find the clang compiler that way). 
Let me confirm we don't need the C++ standard library and remove the sentence if we indeed don't need it.

Reply:
Are we installing these in different places in the docker image? Maybe we should consider co-locating the clang and nvq++ installs.